src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs

```diff
@@ -40,9 +40,8 @@
       {:empty-message "No Workflows"
        :columns [{:header "Data Entity" :starting-width 200
                   :as-text
-                  (fn [workflow]
-                    (let [entity (workflow "workflowEntity")]
-                      (str (entity "entityName") " (" (entity "entityType") ")")))
+                  (fn [workflow] (let [entity (workflow "workflowEntity")] (str (entity "entityName"))))
```

Comment:
This can just be:

    (fn [workflow] (get-in workflow ["workflowEntity" "entityName"]))